### DISCUSSION TITLE
Where is Table Block ?

### DISCUSSION THREAD

**Discussion by MatteoGauthier** [Budibase/budibase#13428] [Help] [answered]
Hey I've seen multiple the `Table Block` component being mentioned, but I don't know where I can find it... Can someone help me ?
<img width="364" alt="Capture d’écran 2024-04-09 à 15 11 25" src="https://github.com/Budibase/budibase/assets/32040951/2fcbc45c-7335-4c4e-92bf-0b1dfc420b68">

I wanted to implement a search bar to search through table rows

**Comment by avivkri** [comment]
Hello @MatteoGauthier,
As per this discussion here, it looks like the table block and the grid block have been removed, consolidated into the updated table component.
https://github.com/Budibase/budibase/discussions/13366

**Reply by MatteoGauthier** [reply]
Thanks for your reply, I get it now

**Comment by ConorWebb96** [comment, marked as answer]
Hey all,

I've updated the documentation. I've removed Grid Block and tagged the table block with deprecated. Additionally, the Grid block information has all been moved into the Table thank you for making us aware of this.

**Reply by MatteoGauthier** [reply]
Thanks for this 🙏
